client/web-sveltekit/src/lib/search/resultsIndicator/SuggestedAction.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+    import { capitalizeFirstLetter } from '$lib/search/utils'
+    import type { Progress, Skipped } from '$lib/shared'
+
+    export let isError: boolean
+    export let progress: Progress
+    export let hasSkippedItems: boolean
+    export let hasSuggestedItems: boolean
+    export let mostSevere: Skipped
+
+    const INTERPUNCT = '\u00B7'
+
+    $: done = progress.done
+</script>
+
+{#if progress}
+    <div class={`action-container ${isError && 'error-text'}`}>
+        <div class="suggested-action">
+            {#if done && !hasSkippedItems}
+                <div class="more-details">See more details</div>
+            {/if}
+
+            {#if done && hasSkippedItems}
+                <div class={`info-badge ${isError && 'error-text'}`}>
+                    {capitalizeFirstLetter(mostSevere?.title ? mostSevere.title : '')}&nbsp;
+                </div>
+            {/if}
+
+            {#if done && hasSuggestedItems}
+                <div class="separator">{INTERPUNCT}</div>
+                <div class="action-badge">
+                    {capitalizeFirstLetter(mostSevere?.suggested ? mostSevere.suggested.title : '')}&nbsp;
+                    <span class="code-font">
+                        {mostSevere.suggested?.queryExpression}
+                    </span>
+                </div>
+            {/if}
+        </div>
+    </div>
+{/if}
+
+<style lang="scss">
+    .action-container {
+        margin-top: 0.3rem;
+    }
+
+    .code-font {
+        background-color: var(--secondary);
+        border-radius: 3px;
+        color: var(--text-body);
+        font-family: var(--code-font-family);
+        font-size: 0.7rem;
+        padding-top: 0.2rem;
+        padding-bottom: 0.2rem;
+        padding-left: 0.2rem;
+        padding-right: 0.2rem;
+    }
+
+    .info-badge {
+        background-color: var(--primary);
+        border-radius: 3px;
+        color: var(--white);
+        padding-left: 0.4rem;
+        padding-right: 0.2rem;
+    }
+
+    .info-badge.duration {
+        background: var(--warning);
+        color: black;
+    }
+
+    .info-badge.error-text {
+        background: var(--danger);
+    }
```

Comment:
You can nest CSS selectors and use `&` to avoid repeating the common class name. E.g. 

```suggestion
    .info-badge {
        background-color: var(--primary);
        border-radius: 3px;
        color: var(--white);
        padding-left: 0.4rem;
        padding-right: 0.2rem;

        &.duration {
            background: var(--warning);
            color: black;
        }

        &.error-text {
            background: var(--danger);
        }
    }
```